File: CRM_API/BO/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace BO
{
    public class Task
    {
        public int TaskNumber { get; set; }
        public int Category { get; set; }
        public int Status { get; set; }
        public string Description { get; set; }
        public string Notes { get; set; }
        public int AssignTo { get; set; }
        public bool NotifyAssignee { get; set; }
        public int Reminder { get; set; }
        public int ContactId { get; set; }
        public DateTime DueDate { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public int ChangedBy { get; set; }
        public DateTime ChangedDate { get; set; }
        public bool IsActive { get; set; }
    }
}

File: CRM_API/CRM.Web/src/app/CRM/models/task.ts
export class Task {
    TaskNumber: number;
    Category: number;
    Status: number;
    Description: string;
    Notes: string;
    AssignTo: number;
    NotifyAssignee: boolean;
    Reminder: number;
    ContactId: number;
    DueDate: Date;
    CreatedBy: number;
    CreatedDate: Date;
    ChangedBy: number;
    ChangedDate: Date;
    IsActive: boolean;
    
    constructor() {
        this.TaskNumber= 0;
        this.Category= 0;
        this.Status= 0;
        this.Description = '';
        this.Notes = '';
        this.AssignTo= 0;
        this.NotifyAssignee= false;
        this.Reminder= 0;
        this.ContactId= 0;
        this.DueDate= new Date();
        this.CreatedBy= 0;
        this.CreatedDate= new Date();
        this.ChangedBy= 0;
        this.ChangedDate= new Date();
        this.IsActive= true;
    }
}